pyproject.toml

```diff
@@ -170,7 +170,7 @@ version = "^0.4.4"
 markers = "sys_platform == 'win32'"
 
 [tool.poetry.dependencies.spacy]
-version = ">=2.1,<2.3"
+version = ">=3.0"
```

Comment:
we should also update the docs [here](If you @joejuzl and @wochinge feel very strongly about this then I will implement this. )

Reply:
I think you tried to copy a link but pasted in a sentence instead, what part of the docs require this?

Reply:
🤦🏻 I mean this https://rasa.com/docs/rasa/installation#dependencies-for-spacy